plugins/modules/openbsd_pkg.py

```diff
@@ -78,6 +78,7 @@
 notes:
   - When used with a C(loop:) each package will be processed individually,
     it is much more efficient to pass the list directly to the O(name) option.
+  - Diff mode is not supported in check mode
```

Comment:
This comment can be removed, the `details` key for the diff mode attribute is a better place for this :)

Reply:
Groovy. Done.